VocaDbModel/Database/Queries/AlbumQueries.cs

```diff
@@ -1046,5 +1046,21 @@ public class AlbumQueries : QueriesBase<IAlbumRepository, Album> {
 
 		}
 
+		public void UpdateVersionVisibility(int archivedVersionId, bool hidden) {

```

Comment:
It would be better to try to reuse this code since it's the same for all object types.